cms_delete_content/templates/delete_confirm.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <template id="delete_confirm" name="CMS default delete confirm">
```

Comment:
JS templates are loaded on demand. Moving it to JS would mean that in your submodules you wouldn't need to call this template in your views. JS would detect the `.cms_delete_content` element, it would bind its events there, and it would load the template only when that element is found, render it only when needed, destroy it when no more needed.